features/waarden.feature

```diff
@@ -189,28 +335,224 @@ Feature: leveren van waarden bij een WOZ-object
         },
         {
-          "vastgesteldeWaarde": 274000,
+          "vastgesteldeWaarde": 266000,
           "waardepeildatum": "2017-01-01"
         }
       ]
       """
+
+    Scenario: meerdere vernietigde beschikkingen met zelfde peildatum en ingangsdatum worden genegeerd
+      Gegeven voor WOZ-object met objectnummer "051800027395" heeft een beschikking met ingangsdatum 01-07-2019
+      En voor een andere belanghebbende is er ook een beschikking met ingangsdatum 01-07-2019
+      En beide beschikkingen met ingangsdatum 01-07-2019 hebben als status "vernietigingbeschikking" (02)
+      En WOZ-object met objectnummer "051800027395" heeft de volgende beschikkingen:
+      | Peildatum  | WOZ-waarde   | Ingangsdatum | Datum beschikking | Status |
+      | 01-01-2019 | 455.000 euro | 01-01-2020   | 13-01-2020        | 01     |
+      | 01-01-2018 | 440.000 euro | 01-07-2019   | 04-09-2019        | 02     |
+      | 01-01-2018 | 440.000 euro | 01-07-2019   | 24-07-2019        | 02     |
+      | 01-01-2018 | 430.000 euro | 01-01-2019   | 14-01-2019        | 01     |
+      | 01-01-2017 | 410.000 euro | 01-01-2018   | 15-01-2018        | 01     |
+      Als het WOZ object wordt opgevraagd met /wozobjecten/051800027395
+      Dan levert de API de volgende waarden:
+      """
+      [
+        {
+          "vastgesteldeWaarde": 455000,
+          "waardepeildatum": "2019-01-01",
+          "ingangsdatum": "2020-01-01",
+          "beschikkingsStatussen": [ "beschikking_genomen" ]
+        },{
+          "vastgesteldeWaarde": 440000,
+          "waardepeildatum": "2018-01-01",
+          "ingangsdatum": "2019-07-01",
+          "beschikkingsStatussen": [ "vernietigingbeschikking", "vernietigingbeschikking" ]
+        },{
+          "vastgesteldeWaarde": 430000,
+          "waardepeildatum": "2018-01-01",
+          "ingangsdatum": "2019-01-01",
+          "beschikkingsStatussen": [ "beschikking_genomen" ]
+        },{
+          "vastgesteldeWaarde": 410000,
+          "waardepeildatum": "2017-01-01",
+          "ingangsdatum": "2018-01-01",
+          "beschikkingsStatussen": [ "beschikking_genomen" ]
+        }
+      ]
+      """
+      En zijn de daaruit bepaalde relevante de waarden:
+      """
+      "waarden": [
+        {
+          "vastgesteldeWaarde": 455000,
+          "waardepeildatum": "2019-01-01"
+        },
+        {
+          "vastgesteldeWaarde": 430000,
+          "waardepeildatum": "2018-01-01"
+        },
+        {
+          "vastgesteldeWaarde": 410000,
+          "waardepeildatum": "2017-01-01"
+        }
+      ]
+      """
+
+    Abstract Scenario: indicatieBezwaarBeroep wanneer een van de statussen bezwaar, (hoger)beroep of cassatie ingesteld maar niet afgehandeld is.
+      Gegeven een beschikking de status "<status>" (<statuscode>) heeft
+      Als het WOZ object wordt opgevraagd
+      Dan levert de API in de betreffende waarde property "beschikkingsStatussen" met de waarde ["status"]
```

Comment:
["status"] moet zijn ["<status>"]. Anders vindt de tekst *status* niet vervangen door bijv. de waarde *bezwaaringediend*

Reply:
ik had bedoeld:
`Dan levert de API in de betreffende waarde property "beschikkingsStatussen" met de waarde ["<status>"]`

zodat de waarde een array is met één item, die gevuld is uit de tabel kolom status. Kan dat ook met [""]?

Reply:
@fsamwel, dat is ook wat ik bedoelde. Ik had de '<' teken, wat ik nu gedaan heb. Anders wordt het gezien als een HTML tag